website/src/content/docs/protocol/filters/date.mdx

```diff
@@ -19,6 +19,11 @@ dateRangeFilter:
       path: "lib/core/lib/core/filters/date.tsp"
       startLine: 24
       endLine: 38
+  python:
+    file:
+      path: "templates/fast-api/src/common_grants/schemas/filters/date_filters.py"
+      startLine: 18
+      endLine: 22
```

Comment:
This doesn't show the full date range filter implementation, just the nested range model.